plugin/slime.vim

```diff
@@ -153,8 +153,8 @@ function! s:VimterminalSend(config, text)
   " Ideally we ought to be able to use a single term_sendkeys call however as
   " of vim 8.0.1203 doing so can cause terminal display issues for longer
   " selections of text.
-  for l in split(a:text,"\n",1)
-    call term_sendkeys(bufnr,l."\<cr>")
+  for l in split(a:text,'\n\zs')
```

Comment:
Vim help:

> \zs     Matches at any position, and sets the start of the match there: The
        next char is the first char of the whole match.

What does it mean for that `split`?

Reply:
split() typically splits text at the regex match, and also removes the matched text.  The \zs modifies the match to start at that point (the end in this case) and thus have zero length so nothing is removed.  The match position is also after the \n so that the effect is that each component of the split will have a \n at the end of it except possibly the very last one which may or may not have a \n at the end depending on the original text.
I agree that the vim help for \zs doesn't make this especially clear, but the help for split() explicitly mentions this usage, which where I discovered it.

Reply:
Nice!